network-metrics-nuttcp.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
It does seem to be in the wrong place (in the root?). I concur, it should probably live in `tests/metrics/network`